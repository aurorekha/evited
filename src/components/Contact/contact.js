import React from 'react';
import './contact.scss';

function Contact() {
  return(
    <div className="contactContainer">
      <h1>Get in touch to discuss your needs</h1>
      <div>
        <h2>Email: macoesltd@gmail.com</h2>
        <h2>Ph: 07550267405</h2>
      </div>
    </div>
  )
}

export default Contact;
