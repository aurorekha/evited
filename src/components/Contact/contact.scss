@font-face {
  font-family: "arapey";
  src: url("../../fonts/arapey/arapey-italic-webfont.woff") format("woff"),
}

.contactContainer {
  font-family: "arapey";
  background: #B3BF6B;
  color: white;
  padding: 2rem;
}
