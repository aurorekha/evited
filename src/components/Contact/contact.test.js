import React from 'react';
import Contact from './contact';
import renderer from 'react-test-renderer';

it('renders Contact section correctly', () => {
  const tree = renderer.create(
    <Contact />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
