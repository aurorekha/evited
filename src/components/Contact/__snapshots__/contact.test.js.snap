// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Contact section correctly 1`] = `
<div
  className="contactContainer"
>
  <h1>
    Get in touch to discuss your needs
  </h1>
  <div>
    <h2>
      Email: macoesltd@gmail.com
    </h2>
    <h2>
      Ph: 07550267405
    </h2>
  </div>
</div>
`;
